python/pyproject.toml

```diff
@@ -112,8 +107,15 @@ skip = [
     "__init__.py",
 ]
 
-[tool.setuptools]
-license-files = ["LICENSE"]
+[tool.scikit-build]
+build-dir = "build/{wheel_tag}"
+cmake.build-type = "Release"
+cmake.minimum-version = "3.26.4"
+ninja.make-fallback = true
+sdist.reproducible = true
```

Comment:
Do you know what this flag does? I read the docs but I don't fully understand its purpose. https://scikit-build-core.readthedocs.io/en/latest/api/scikit_build_core.settings.html#scikit_build_core.settings.skbuild_model.SDistSettings.reproducible

Reply:
Yeah, it basically guarantees binary equivalence between two sdists produced from the same source. In particular, it will normalize things like file permissions (in tarballs, not the underlying files) and timestamps in a way that is agnostic to the current build environment. It ensures that if you build an sdist at two different times but with identical code you get binary-equivalent sdists.